alttpo/SettingsWindow.as

```diff
@@ -809,5 +809,11 @@ class SettingsWindow {
 
     // persist settings to disk:
     save();
+    
+    if(settings.started)
+    {
+      disconnect();
+      connect();
+    }
```

Comment:
Oh we definitely don't want a disconnect/reconnect just because color is changed.